docs/execute.rst

```diff
@@ -27,13 +27,13 @@ Example
 Let's start with a complete quick example, leaving detailed explanations
 to the following sections.
 
-**Import**: First we import nbclient and the :class:`Executor`
+**Import**: First we import nbformat and the :class:`Executor`
 class::
 
-    import nbclient
+    import nbformat
     from nbclient.execute import Executor
```

Comment:
Does the `from...` line need to change, too?

Reply:
No.